pkg/registry/core/node/strategy.go

```diff
@@ -170,20 +153,7 @@ func (nodeStatusStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime
 	oldNode := old.(*api.Node)
 	newNode.Spec = oldNode.Spec
 
-	if !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) && !nodeStatusConfigInUse(oldNode) {
```

Comment:
shouldn't this be `if !nodeStatusConfigInUse(oldNode) {`?

Reply:
done